app/models/team.rb

```diff
@@ -138,6 +138,54 @@ def update_captain_and_eligibility(*)
     update_eligibility
   end
 
+  def generate_certs(rank, total, send_email) # iterate over all users to create certs and send ranking email
```

Comment:
`generate_completion_certificates` might be a better name for this, generate_certs makes me thing its their openvpn cert